glue_jupyter/common/state_widgets/viewer_histogram.py

```diff
@@ -1,47 +1,36 @@
 import ipyvuetify as v
-
-from ...widgets.linked_dropdown import LinkedDropdownVuetify
-from .vuetify_helpers import link_vuetify_checkbox, link_vuetify_generic
+import traitlets
+from ...vuetify_helpers import load_template, link_glue
+from ...widgets.linked_dropdown import get_choices
 
 __all__ = ['HistogramViewerStateWidget']
 
 
-def model_to_bool(x):
-    return x == 0
-
+class HistogramViewerStateWidget(v.VuetifyTemplate):
+    template = load_template('viewer_histogram.vue', __file__)
+    x_axis = traitlets.List().tag(sync=True)
+    selected_axes = traitlets.Unicode().tag(sync=True)
+    normalize = traitlets.Bool().tag(sync=True)
+    cumulative = traitlets.Bool().tag(sync=True)
+    show_axes = traitlets.Bool().tag(sync=True)
```

Comment:
I wonder whether we could avoid having to explicitly create traitlets for all of the attributes - since this duplicates information in the state class? Is there any way we could link the template directly to the state class? Or if not, can we auto-generate the traitlets, perhaps using a meta-class?